controllers/profiles/object_creators.go

```diff
@@ -205,9 +205,32 @@ func defaultServiceCreator(workflow *operatorapi.KogitoServerlessWorkflow) (clie
 			}},
 		},
 	}
+
 	return service, nil
 }
 
+// defaultNetworkCreator is an objectCreator for a basic Route for a workflow using dev profile
+// running on OpenShift.
+// It enables the exposition of the dev service using an OpenShift Route.
+// See: https://github.com/openshift/api/blob/d170fcdc0fa638b664e4f35f2daf753cb4afe36b/route/v1/route.crd.yaml
+func defaultNetworkCreator(workflow *operatorapi.KogitoServerlessWorkflow) (client.Object, error) {
```

Comment:
Maybe it is better to move this func to the 'utils/openshift' package and rename this func to ExposeRoute or something similar.